### DISCUSSION TITLE
Q: Why Honor goes down?

### DISCUSSION THREAD

**Discussion by Rami-0** [codewars/codewars.com#2768] [General]
hey, i would like to know in which cases the honor goes down? if anyone had clue

**Comment by Madjosz** [comment]
If it is only small amounts then it is because someone downvoted one of your comments in discourse, removed their upvotes to one of your solutions (best practice/clever) or you removed a kata satisfaction vote yourself.
If it is bigger amounts then most likely some of your solutions have been marked as cheating/plagiarism.
If your honor went back to something around 0 then your account got suspended because of exessive cheating or copying solutions to purple kata en masse from Github.

**Reply by hobovsky** [reply]
Another uncommon possibility is when a kata you solved goes back to beta and/or gets reapproved with a lower rank.
